uint32 motorIndex
bool relativeMode
---
